react-demo-app/src/components/Softphone.tsx

```diff
@@ -0,0 +1,67 @@
+import { FormEvent, useState } from "react";
+import Card from "./Card";
+import { GuxButton, GuxToggle } from "genesys-spark-components-react";
+import "./Softphone.css";
+import useSdk from "../hooks/useSdk";
+import PendingSessionsTable from "./PendingSessionsTable";
+import ActiveConversationsTable from "./ActiveConversationsTable";
+import HandledPendingSessionsTable from "./HandledPendingSessionsTable";
+import StationDetails from "./StationDetails";
+
+export default function Softphone() {
+  const [phoneNumber, setPhoneNumber] = useState("*86");
+  const [onQueueStatus, setOnQueueStatus] = useState(false);
+  const { startSoftphoneSession, updateOnQueueStatus } = useSdk();
+
+  function placeCall(event?: FormEvent<HTMLFormElement>) {
+    if (event) {
+      event.preventDefault()
+    }
+    startSoftphoneSession(phoneNumber);
+  }
+  function toggleQueueStatus() {
+    updateOnQueueStatus(!onQueueStatus);
+    setOnQueueStatus(!onQueueStatus);
+  }
+
+  // GuxButton's don't support form events currently but we want a form for keyboard navigation.
```

Comment:
UPDATE: I tried using the `GuxFormBeta` component but its in beta and is still pretty wonky and doesn't support form events from `GuxButton` so for now we're just going to hack this and have a normal form call the same function as the button. The main purpose to this is to be able to place calls by simply hitting enter.